packages/affiliates/payouts/devmining/2021-01-04_2021-01-11.json

```diff
@@ -0,0 +1,89 @@
+{
+  "config": {
+    "empWhitelist": [
+      ["0x3a93E863cb3adc5910E6cea4d51f132E8666654F", "0x53911776641d6dF38B88b9eF27f920c617E3Cb5e"],
+      ["0x516f595978D87B67401DaB7AfD8555c3d28a3Af4", "0x97990B693835da58A281636296D2Bf02787DEa17"],
+      ["0x4AA79c00240a2094Ff3fa6CF7c67f521f32D84a2", "0x7eb1e51351380E67F641B84545bf41eCdc10298E"],
+      ["0xf32219331A03D99C98Adf96D43cc312353003531", "0x37a0b32e07CD0b53AFb2981Cf02A74e10B80B577"],
+      ["0x1c3f1A342c8D9591D9759220d114C685FD1cF6b8", "0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35"],
+      ["0xE4256C47a3b27a969F25de8BEf44eCA5F2552bD5", "0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35"],
+      ["0xeAddB6AD65dcA45aC3bB32f88324897270DA0387", "0xF53FBC3aDbe91E1A408716C0AF4C0cDC17A2cca8"],
+      ["0xf215778f3a5e7ab6a832e71d87267dd9a9ab0037", "0xb8E237CC8D2E81b7C9C9C4B5877200f85100fC87"],
+      ["0x267D46e71764ABaa5a0dD45260f95D9c8d5b8195", "0x531B290230b1809d0AE29Ea834EdF96748F49Bf9"],
```

Comment:
This EMP is counted twice in the whitelist and fallbackPrices. Not sure if it affected the rewards amounts though

Reply:
thanks good catch.